modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/ElasticsearchContainer.java

```diff
@@ -0,0 +1,65 @@
+package org.testcontainers.elasticsearch;
+
+import org.apache.http.HttpHost;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.Base58;
+
+import java.time.Duration;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static java.net.HttpURLConnection.HTTP_UNAUTHORIZED;
+
+/**
+ * Represents an elasticsearch docker instance which exposes by default port 9200 and 9300 (transport.tcp.port)
+ * The docker image is by default fetch from docker.elastic.co/elasticsearch/elasticsearch
+ * @author dadoonet
+ */
+public class ElasticsearchContainer extends GenericContainer {
+
+    /**
+     * Elasticsearch Default HTTP port
+     */
+    private static final int ELASTICSEARCH_DEFAULT_PORT = 9200;
+
+    /**
+     * Elasticsearch Default Transport port
+     */
+    private static final int ELASTICSEARCH_DEFAULT_TCP_PORT = 9300;
+
+    /**
+     * Elasticsearch Docker base URL
+     */
+    private static final String ELASTICSEARCH_DEFAULT_BASE_URL = "docker.elastic.co/elasticsearch/elasticsearch";
```

Comment:
Could we change the name of this constant to `ELASTICSEARCH_DEFAULT_IMAGE` or similar?